tests/foreman/cli/test_remoteexecution.py

```diff
@@ -101,8 +100,8 @@ class TestRemoteExecution:
     @pytest.mark.tier3
     @pytest.mark.pit_client
     @pytest.mark.pit_server
-    @pytest.mark.parametrize('fixture_vmsetup', [{'nick': 'rhel7'}], ids=['rhel7'], indirect=True)
-    def test_positive_run_default_job_template_by_ip(self, fixture_vmsetup):
+    @pytest.mark.rhel_ver_list([7])
```

Comment:
same question about the support matrix. my suggestion would be to use `[7,8,9]` at least